bin/decorate

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# bin/decorate <build-dir>
+
+BIN_DIR=$(cd $(dirname $0); pwd)
+ROOT_DIR=$(dirname $BIN_DIR)
+BUILD_DIR=$1
+
+if [ -d $BUILD_DIR/.signalfx ]
+then
+  echo 'signalfx'
+  exit 0
+else
+  exit 1
+fi
```

Comment:
Tells meta-buildpack whether this decorator applies based on whether they have signalfx config in their app dir.